operations/sat/sat_in_csm.md

```diff
@@ -0,0 +1,59 @@
+# System Admin Toolkit (SAT) in CSM
+
+The System Admin Toolkit (SAT) is a command-line interface that can assist administrators with common tasks, such as
+troubleshooting and querying information about the HPE Cray EX System, system boot and shutdown, and replacing hardware
+components. In CSM 1.3 and newer, the `sat` command is available on the Kubernetes NCNs without installing the SAT
+product stream.
+
+## SAT Product Stream Components
```

Comment:
Currently in the docs-csm repository we follow the HPE style guide, which (in the latest version we've seen) specifies that headings (other than the main page title) should use sentence casing (that is, the way you would capitalize a normal sentence).

```suggestion
## SAT product stream components
```

Reply:
Fixed in 14bd2c494e86e31f7460bf914b6d6cc49475dae4